contracts/Bridge.sol

```diff
@@ -136,26 +141,31 @@ contract Bridge is RelayerRole, Pausable {
                 : DepositStatus.Rejected;
         }
 
-        CrossTransferStatus storage crossStatus = crossTransferStatuses[batchNonceElrondETH];
+        CrossTransferStatus storage crossStatus = crossTransferStatuses[batchNonceMvx];
         crossStatus.statuses = statuses;
         crossStatus.createdBlockNumber = block.number;
     }
 
     /**
         @notice Only returns values if the cross transfers were executed some predefined time ago to ensure finality
-        @param batchNonceElrondETH Nonce for the batch
+        @param batchNonceMvx Nonce for the batch
         @return a list of statuses for each transfer in the batch provided
      */
-    function getStatusesAfterExecution(uint256 batchNonceElrondETH) external view returns (DepositStatus[] memory) {
-        CrossTransferStatus memory crossStatus = crossTransferStatuses[batchNonceElrondETH];
-        require(crossStatus.createdBlockNumber > 0, "Invalid nonce requested");
-        require((crossStatus.createdBlockNumber + batchSettleBlockCount) <= block.number, "Statuses not final yet");
+    function getStatusesAfterExecution(uint256 batchNonceMvx) external view returns (DepositStatus[] memory, bool isFinal) {
+        CrossTransferStatus memory crossStatus = crossTransferStatuses[batchNonceMvx];
+        return (crossStatus.statuses, _isMvxBatchFinal(crossStatus.createdBlockNumber));
+    }
```

Comment:
This method has been updated to return statuses even if the execution is not considered final (has not reached finality).

Thus it has some implications:

1. The comment has not been updated to reflect this new behavior.
```
@notice Only returns values if the cross transfers were executed some predefined time ago to ensure finality
```

2. Other systems interacting with the contract, will need to be updated to always check the second returned argument `isFinal == true` before considering the list of transfers final. Otherwise, none of the items in the list of statuses should be considered final.
3. Other systems will need to be upgraded to always expect a result since the previous behavior was sometimes reverting (because of the `require`s)

Reply:
cached the descriptions to make it clear